src/components/Books.js

```diff
@@ -9,10 +10,20 @@ function Books() {
         flexDirection: 'row',
         justifyContent: 'center',
         alignItems: 'center',
+        flex: 1,
       }}
       >
-        <section style={{ display: 'flex', flexDirection: 'column' }}>
+        <section
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            flex: 1,
+            boxSizing: 'border-box',
+            padding: '4%',
+          }}
+        >
           <BooksList />
+          <div className={styles.horizontalDivider}> </div>
           <NewBook />
         </section>
```

Comment:
- [ ] Thank you for your effort to match the given design. you are almost there. Let's update section to add a new book.
- Your design:
![new_book1](https://user-images.githubusercontent.com/65860678/222806708-07c9beb2-d6e2-4faa-8a7e-b117e824df2d.PNG)
- Given design
![new_book2](https://user-images.githubusercontent.com/65860678/222806755-8b2b8684-3362-44cc-bb5c-f142c120a556.PNG)